### ISSUE TITLE
Shader & Resource pack management

### ISSUE THREAD

**Issue by Robinerra** [gorilla-devs/ferium#141]
Support for managing resource packs and shaders would be extremely helpful and in my opinion would fit in with what Ferium does. 

This would add some complexity to setting up a profile however, from having to specify the shader and pack output folders, but making it optional and only required once the feature is used would be a godo compromise.

**Comment by theRookieCoder** [comment]
This would fit in more with v5, where ferium would ideally become an entire launcher. For now ferium remains a mod manager and this issue will be in the backlog, it will unfortunately not be a priority for now

**Comment by Aces-and-Jacks** [comment]
> where ferium would ideally become an entire launcher.

By launcher do you mean it would become it's own application? Cause if so I like it better as more of a terminal thing

**Comment by theRookieCoder** [comment]
I had initially been thinking of an alternative GUI frontend, but I think it's better to purely focus on the CLI experience. So yeah a launcher expansion would most likely be CLI only.

**Comment by Aces-and-Jacks** [comment]
If/when cerium becomes a launcher would it still work as purely just a mod manager?

**Comment by theRookieCoder** [comment]
If you're asking if you'd be able to use it only for mod management, and use something else for launching the game, then yeah absolutely.

**Comment by Aces-and-Jacks** [comment]
If/whenever you decide to add it do you know if it would work similarly for resource packs and shaders as it does for mods? Or if there's going to be a complete rewrite of how it works? I'd get it if it's too distant in the future to tell yet

Also if mod management would work the same way it does now or if that'll change?

**Comment by theRookieCoder** [comment]
Yes, mod management would work the same way, and as would any other project types too like resource packs and shaders. There most likely won't be a rewrite of the mod management.

**Comment by Aces-and-Jacks** [comment]
> If/whenever you decide to add it do you know if it would work similarly for resource packs and shaders as it does for mods? 

Whenever you do decide to add support for resource packs you might want to seperate it by pack type if you can. Since different versions of minecraft have different pack types. Similar to how you have mods for different versions of minecraft. I wasn't sure if this had been considered

**Comment by theRookieCoder** [comment]
> Whenever you do decide to add support for resource packs you might want to seperate it by pack type if you can.

Do you mean there should be Minecraft version filtering for other types of projects too? Or do you mean other types of projects should have different filters configurable? I'm not sure yet, but I do plan on having both.
